source/NVDAObjects/UIA/__init__.py

```diff
@@ -2265,15 +2290,9 @@ def event_stateChange(self):
 
 class SearchField(EditableTextWithSuggestions, UIA):
 	"""An edit field that presents suggestions based on a search term.
+	This is now an empty class as Functionality has been moved to the base EditableText behaviour.
 	"""
 
-	def event_UIA_controllerFor(self):

```

Comment:
does removing this function break add-on compatibility?

Reply:
No, as it has been moved to the base.